NEWS.txt

```diff
@@ -3,282 +3,204 @@ Release Notes
- - optimizer=fast_compile moves computation to the GPU
- - Raise the user stack trace more frequently.
- - Change dev version numbering to follow the PEP 440
-
-
-New Interface (reuses existing functionality):
- - theano.tensor.nnet.relu
- - theano.tensor.nnet.elu
- - BatchNormalization.
- - MaxAndArgmax support axis=None
- - Add theano.tensor.compress (equivalent of numpy.compress)
- - theano.tensor.signal.downsamples.max_pool_2d_same_size
- - COp
- - __props__
-
-New features
- - tensor.unique
- - map_variables
- - erfcx
- - mgrid, ogrid
- - allclose
- - BreakpointOp
- - Make bincount work on GPU
- - SolveOp on GPU
- - Optional optimization remove_all_assert
- - AllocEmpty
- - LogSoftmax, for stability optimization when the crossentropy optimization does not apply.
- - theano.tensor.repeat works on GPU
- - BatchedDot on the GPU and faster on the CPU.
- - Faster batched_tensordot and make it work on GPU.
- - SoftmaxGrad grad
- - 3d conv via CorrMM on the GPU
- - CPU Max Pool support of padding and strides!=windows size
- - theano.function() now accepts a dict for the outputs. When doing this, the function will return a dict. Helpful to keep track of which output is what.
- - Warn for unknown or misspelled theano config variables
- - theano.tensor.tile update (accept symbolic reps, work on GPU)
- - scan how have a strict flag. If set to True, this make scan building faster and could make execution faster.
- - theano.tensor.signal.conv2d(2d,2d) output 2d answer
- - More convolution parameter supported
- - Bilinear interpolation of images
-
-
-Speed-ups:
- - Faster SetSubtensor on the GPU.
- - Support more reduction pattern on the GPU.
- - More graph optimization
- - Faster graph optimization
- - GpuCrossentropySoftmaxArgmax1HotWithBias
-
-
-Crash/no return fixes:
- - Fix crash in the assert op grad
- - Fix curand crash on Mac
- - Multiple Fix scan crashes
- - Finish to update all Op.grad() implementation to the new interface
+Theano 0.9.0beta1 (24th of January, 2017)
+=========================================
+
+This release contains a lot of bug fixes and improvements + new features, to prepare the upcoming release candidate.
+
+Highlight:
+ - Many computation and compilation speed up
+ - More numerical stability by default for some graph
+ - Jenkins (gpu tests run on PR in addition to daily buildbot)
+ - Better handling of corner cases for theano functions and graph optimizations
+ - More graph optimization (faster execution and smaller graph, so more readable)
+ - Less c code compilation
+ - Better Python 3.5 support
+ - Better numpy 1.12.0rc2 support
+ - Support newer Mac and Windows version
+ - Conda packages for Mac, Linux and Windows
+ - Theano scripts now works on Windows
+ - scan with checkpoint (trade off between speed and memory usage, useful for long sequences)
+ - Added a bool dtype
+ - Added a deprecation message for the old backend.
+
+ - New back-end:
+   - float16 storage
+   - print pci bus id
```

Comment:
I suggest: `better mapping between theano device number and nvidia-smi number, using the PCI bus ID of graphic cards`